packages/html/ds/src/button/button.html

```diff
@@ -0,0 +1,18 @@
+{% from 'button/helpers.html' import getVariantAppearanceClass, getSizeClass, getIconPosition, isButtonDisabled %}
+{% from 'icon/icon.html' import govieIcon %}
+
+{% macro govieButton(props) %}
+
+{% set appearance = props.appearance or 'default' %}
+{% set variant = props.variant or 'primary' %}
+{% set size = props.size or 'medium' %}
+{% set isDisabled = 'disabled' if props.disabled else 'notDisabled' %}
+
+<button data-testid="govieButton-{{ appearance }}-{{ variant }}-{{ size }}-{{ isDisabled }}" class="gi-gap-2 gi-flex gi-rounded-sm focus:gi-outline focus:gi-outline-[3px] focus:gi-outline-yellow-400 focus:gi-outline-offset-0 gi-items-center {{ getVariantAppearanceClass(props.disabled, props.variant, props.appearance) | trim }} {{getSizeClass(props.size) | trim }} {{getIconPosition(props.icon.position) | trim }} {{isButtonDisabled(props.disabled, props.variant, props.appearance) | trim }}">
```

Comment:
Please, add these classes to `packages/design/tailwind/css/components.css` in a new class named, for example, `gi-button`.
Add the variants as separate classes to reuse on React and plain HTML.

Reply:
Added the classes